src/parser/src/main/java/io/github/syst3ms/skriptparser/sections/SecMap.java

```diff
@@ -0,0 +1,62 @@
+package io.github.syst3ms.skriptparser.sections;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.Statement;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.Variable;
+import io.github.syst3ms.skriptparser.lang.lambda.ReturnSection;
+import io.github.syst3ms.skriptparser.lang.lambda.SkriptFunction;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.types.changers.ChangeMode;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public class SecMap extends ReturnSection<Object> {
```

Comment:
Consider adding some Javadoc. Applies to all previous sections as well of course.